ui/src/app/utils/access-utils.spec.tsx

```diff
@@ -738,7 +738,7 @@ describe('getAccessModuleStatusByName', () => {
       100,
     ],
     [
-      AccessModule.RASLINKLOGINGOV,
+      AccessModule.RASLINKIDME,
       AccessModule.RASLINKIDME,
```

Comment:
it seems duplicates the next line?

Reply:
The first is the inputted access module and the second is the expected output access module. Those could be the same value or different depending on whether ID.me or Login.gov was completed most recently. 

I am in the process of rewriting these tests to make them more readable.